migration/src/m0000475_improve_version_comparison_fns/semver_cmp.sql

```diff
@@ -0,0 +1,128 @@
+
+create or replace function semver_cmp(left_p text, right_p text)
```

Comment:
it is unclear if this function covers off all of http://ftp.rpm.org/api/4.4.2.2/rpmvercmp_8c-source.html .... I did a version in sql here https://github.com/RedHatProductSecurity/component-registry/blob/main/corgi/core/migrations/0092_install_stored_proc.py ... if I have the time tmrw will cross reference between rpmvercmp and this

Reply:
and yes I know this comment is expanding scope ;)

Reply:
So far only semver cmp. Need to do *many more*.

Reply:
And I'll definitely steal your RPM version cmp.

Reply:
+1 - we assumed rpm folks have encountered every variation and stole from them thusly ... I think my sql version is correct eg. ran it against their test cases (but did not embed in corgi)